### PR TITLE
CDAP-5652 CLI command to get Workflow node states.

### PR DESCRIPTION
**Pull request by sagarkapare**
JIRA: https://issues.cask.co/browse/CDAP-5652

Sample command output:
```java
cdap (http://localhost:10000/namespace:default)> get workflow state PurchaseHistory.PurchaseHistoryWorkflow b48355a2-0599-11e6-8ace-da6a50dd7318
+============================================================================================+
| node id                | node status | runid                                | failurecause |
+============================================================================================+
| PurchaseHistoryBuilder | COMPLETED   | b4b1b8a3-0599-11e6-9e55-da6a50dd7318 |              |
+============================================================================================+
```

### CODE REVIEW

**Comment by sagarkapare** [comment]
Release build is running here - http://builds.cask.co/browse/CDAP-DUT3982-1

**Comment by gsps1** [comment]
this will need docs changes right? can you add that as well in this PR? 

Also, can you add a preview of how the command will look like and the returned result in the PR description?

File: cdap-cli/src/main/java/co/cask/cdap/cli/command/GetWorkflowNodeStatesCommand.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                                                 ProgramType.WORKFLOW, programIdParts[1],
+                                                 arguments.get(ArgumentName.RUN_ID.toString()));
+
+    Table table = getWorkflowNodeStates(programRunId);
+    cliConfig.getTableRenderer().render(cliConfig, printStream, table);
+  }
+
+  @Override
+  public String getPattern() {
+    return String.format("get workflow node states <%s> <%s>", elementType.getArgumentName(), ArgumentName.RUN_ID);
```
Comment (gsps1) [inline cdap-cli/src/main/java/co/cask/cdap/cli/command/GetWorkflowNodeStatesCommand.java]: the handler api is like `nodes/state` while its `node states` here.

**Comment by sagarkapare** [comment]
@gsps1 documentation for the commands is generated based on the help. I added sample command and its output to the PR description.

**Comment by gsps1** [comment]
lgtm

**Comment by sagarkapare** [comment]
Thanks @gsps1 for the review. Build is green here - http://builds.cask.co/browse/CDAP-DUT3982-2 after the first commit. Second commit simply changed the description and RAT passed. So squashing and Merging.

**Pull request merged**